backend/routes.js

```diff
@@ -258,7 +267,37 @@ export default class Servers {
                         status = 201;
                     }
                     saveSetting('USERS', existingUsers);
-                    res.json(status, {});
```

Comment:
Does it make sense to move the `saveSetting('USERS', existingUsers)` to after the user has been validated?

Reply:
Yep, make sense to me.. Updated !!